New Feature: Events Manager. Some shortcodes were not included due to it...

oherman opened the pull request:
... being ineffective in page design, or is otherwise redundant.

bfintal commented on lib/shortcode/events-manager.php:
```diff
@@ -0,0 +1,169 @@
+		return;
+	} 
+	
+	add_editor_style( EM_DIR_URI . 'includes/css/events_manager.css' );
```
Add check for `defined( 'EM_DIR_URI' )`

bfintal commented on lib/shortcode/events-manager.php:
```diff
@@ -0,0 +1,169 @@
+
+	// Register Shortcake UI for Events Manager Events List
+	shortcode_ui_register_for_shortcode(
+		'events_list',
```
There are 2 declarations for the shortcode `events_list`. Is the earlier one needed?

bfintal commented on lib/shortcode/events-manager.php:
```diff
@@ -0,0 +1,169 @@
+	shortcode_ui_register_for_shortcode(
+		'events_list',
+		array(
+			'label' => __( 'Events Manager - Events List', 'pbsandwich' ),
```
Remove the dash in the name of the shortcode since all our other third-party shortcodes do not have any dashes

bfintal commented:
@oherman after you do all the changes please comment about and let @Gambit-Alyssa know when it's ready for testing

oherman commented:
Changes done. Dupes spun off into a new function and requested changes are done.

@Gambit-Alyssa get ready to rip this apart.

> On Mar 18, 2015, at 1:04 PM, Benjamin Intal <notifications@github.com> wrote:
> 
> @oherman <https://github.com/oherman> after you do all the changes please comment about and let @Gambit-Alyssa <https://github.com/Gambit-Alyssa> know when it's ready for testing
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/gambitph/Page-Builder-Sandwich/pull/110#issuecomment-82741900>.
>

Gambit-Alyssa commented:
Events Manager: Events Submission Form
It shows differently in backend and frontend.

Backend View: http://prntscr.com/6in6el
Frontend View:
http://prntscr.com/6in7a5
http://prntscr.com/6in7eq

Gambit-Alyssa commented:
Element's Text can be highlighted and moved around, moving the text makes the whole element go "poof".

Screencast: http://screencast-o-matic.com/watch/coeqXfesul

oherman commented:
Different configs yield different styling. This might be a theme-dependent issue. Please try out with various theme and see if the same results occur.

> On Mar 19, 2015, at 6:57 PM, Alyssa M <notifications@github.com> wrote:
> 
> Events Manager: Events Submission Form
> It shows differently in backend and frontend.
> 
> Backend View: http://prntscr.com/6in6el <http://prntscr.com/6in6el>
> Frontend View:
> http://prntscr.com/6in7a5 <http://prntscr.com/6in7a5>
> http://prntscr.com/6in7eq <http://prntscr.com/6in7eq>
> —
> Reply to this email directly or view it on GitHub <https://github.com/gambitph/Page-Builder-Sandwich/pull/110#issuecomment-83505017>.
>

oherman commented:
Addressed by commit ec02d32

> On Mar 19, 2015, at 7:05 PM, Alyssa M <notifications@github.com> wrote:
> 
> Element's Text can be highlighted and moved around, moving the text makes the whole element go "poof".
> 
> Screencast: http://screencast-o-matic.com/watch/coeqXfesul <http://screencast-o-matic.com/watch/coeqXfesul>
> —
> Reply to this email directly or view it on GitHub <https://github.com/gambitph/Page-Builder-Sandwich/pull/110#issuecomment-83508832>.
>
